vsg/styles/minimal.yaml

```diff
@@ -0,0 +1,4 @@
+# The name must be a single word
+name : minimal
```

Comment:
`minimal` style is not mentioned in the documentation. Thee are `base`, `disabled`, `legacy`. I would remove `base` and `disabled`, and release only `minimal` and `legacy` in the 2.0.0. release. Maybe it would be good to rename `minimal` to `indent`/`indent_only`?

Reply:
I think `indent_only` makes sense.  I still not sure about removing `base`.  I will add documentation for `indent_only`.